.evergreen/config.yml

```diff
@@ -0,0 +1,47 @@
+functions:
+  fetch source:
+    # Executes git clone
+    - command: git.get_project
+      params:
+        directory: src
+    # Applies the submitted patch, if any
+    - command: git.apply_patch
+
+  exec script:
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: src
+        shell: bash
+        script: |
+          ${PREPARE_SHELL}
```

Comment:
Doesn't this needs to be defined somewhere? I'm surprised this doesn't cause the script to error.

Reply:
I removed it for currently it is not defined.
Guess shell might interpret unknown variable as empty string. For instance, the following command:
```
# echo $UNKNOWN_VARIABLE
```
simply output an empty string without complaining, unless `set -u` is run first.